.gitignore

```diff
@@ -1,2 +1,3 @@
 node_modules
 .env
+google.keys.json
```

Comment:
If this is needed, add a notice at the readme or put an empty google.keys.json file

Reply:
Adding example